### PR TITLE
feat: allow to choose UNLICENSED as license

### LINKED ISSUE #22: allow to use UNLICENSED for license
I would like to create private apps using this tool, especially with [vivliostyle/create-book](https://github.com/vivliostyle/create-book).

In [documents of npm](https://docs.npmjs.com/files/package.json), we can use `{ "license": "UNLICENSED" }` for private packages.  So I think it would be a good idea to support this specification in create-create-app as well.

Consideration:

* Should we support this feature in default, or using `extra` arguments in `create()`?
* When choosing UNLICENSED, should we also add `{"private": true}` in package.json?

### PR DESCRIPTION
**Pull request by takahashim**
In package.json, when your package is private, you can use `{"license":"UNLICENSED"}`
https://docs.npmjs.com/files/package.json#license

fix #22

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -265,14 +265,18 @@ export async function create(appName: string, options: Options) {
...
+    const license = makeLicenseSync(args.license, {
+      year,
+      project: name,
+      description: args.description,
+      organization: getContact(args.author, args.email),
+    })
+    const licenseText = license.header + license.text + license.warranty
+    fs.writeFileSync(path.resolve(packageDir, 'LICENSE'), licenseText)
+  } catch (e) {
+    // do not generate LICENSE
```
Comment (uetchy) [inline src/index.ts:278]: What kind of errors are you expecting from this block?

**Review by uetchy — COMMENTED** [review]
Thanks @takahashim! I left a tiny question; everything else is simply great.

File: src/index.ts (hunk 1 above)
Comment (takahashim) [reply src/index.ts:278]: `makeLicenseSync()` throws "License not found" error when unexpected license is chosen https://github.com/sergeysova/license.js/blob/1b0b605cd895230df54c15e24120cecb59c82263/lib/index.js#L12-L14
`UNLICENSED` is "unexpected license" for license.js npm. https://github.com/sergeysova/license.js/blob/master/licenses/index.json

**Comment by uetchy** [comment]
Merged🚀 Thank you @takahashim!

**Pull request merged**